defaults/main.yml

```diff
@@ -38,18 +40,24 @@ cartridge_role_scenarios:
     - configure_instance
     - restart_instance
     - wait_instance_started
+    - wait_all_instances_alive
+    - wait_cluster_has_no_issues
```

Comment:
Are you sure that this tasks will be works here?

Reply:
`wait_cluster_has_no_issues` will work.